flycheck.el

```diff
@@ -4170,7 +4193,14 @@ non-nil."
     (setq flycheck-display-error-at-point-timer
           (run-at-time flycheck-display-errors-delay nil 'flycheck-display-error-at-point))))
 
-
+(defun flycheck-try-fix-error-at-point ()
+  "Attempt to fix error at point using the list of fixer functions."
+  (interactive)
+  (let ((err (car-safe (flycheck-overlay-errors-at (point)))))
+    (if err
+        (run-hook-with-args-until-success 'flycheck-fix-error-functions err)
+      (message "No errors at point to fix!"))))
```

Comment:
I'm not sure whether it's good to just fix the first error at point.  In most cases there'll only be one error at point, for sure, but what if there are two and the first one isn't fixable whereas the second one is?  We always order overlays by error level so the user will never get a chance to fix the second error automatically.